corehq/apps/domain/templates/domain/internal_calculations.html

```diff
@@ -39,7 +39,7 @@
     </script>
 {% endblock %}
 
-{% block main_column %}
+{% block page_content %}
     <div class="form form-horizontal">
```

Comment:
This form is pretty unusual. It's even not using <form> tag and it's only used to show data on UI not to send something to backend so for me it doesn't make sense to convert this to crispy.